components/authz-service/storage/v2/conditionattribute.go

```diff
@@ -0,0 +1,76 @@
+package v2
+
+import (
+	"fmt"
+	"strings"
+)
+
+// ConditionAttribute is an enum of attributes a project rule condition can be.
+type ConditionAttribute int
+
+const (
+	Organization ConditionAttribute = iota
+	ChefServer
+	Environment
+	ChefRole
+	ChefTag
+	PolicyName
+	PolicyGroup
+)
+
+func (c ConditionAttribute) String() string {
+	if c < Organization || c > PolicyGroup {
+		panic(fmt.Sprintf("unknown value from iota ConditionAttribute on String() conversion: %d", c))
```

Comment:
This can never happen. Let's remove L22-24.

*Update*: [you can make it happen](https://play.golang.org/p/_lF3iyHow3B). And that fact makes me sad.